cmd/bridge/main.go

```diff
@@ -335,20 +300,10 @@ func main() {
 		k8sAuthServiceAccountBearerToken string
 	)
 
-	var secureCookies bool
-	if baseURL.Scheme == "https" {
-		secureCookies = true
-		klog.Info("cookies are secure!")
-	} else {
-		secureCookies = false
-		klog.Warning("cookies are not secure because base-address is not https!")
-	}
-
```

Comment:
Why is this removed?

Reply:
this is `useSecureCookies = baseURL.Scheme == "https"` in `getAuthenticator()`